rust/tests/tests/helpers/otel_json.rs

```diff
@@ -0,0 +1,30 @@
+use serde_json::Value;
+
+pub fn attribute_of_first_span(trace: &Value, attribute: String) -> Option<String> {
+    if let Some(span) = spans(trace).first() {
+        let value = span[attribute].as_str().unwrap_or_default();
+        return Some(value.to_string());
+    }
+    None
+}
+
+pub fn ids_and_parent_span_ids(trace: &Value) -> (Vec<String>, Vec<String>) {
```

Comment:
This function is actually used in the tests, I'm not sure why GH isn't picking up on that. I'll look into it.

Reply:
i don't think rust counts using something in a test as being used in the exported code. Not a big deal but one way to get rid of it would be to move the definition to the tests as well. but if that's unreasonable then i think it's fine.

Reply:
All of the function in this module are only used by the tests, it's weird that only this one lights up the warning. I've suppressed the warnings for that module in [7b7184c](https://github.com/dylibso/observe-sdk/pull/17/commits/7b7184c8e8356427c768149eab51c1fe0173f93a) since it's incorrect.